lib/src/main/java/ca/jbrains/pos/stub/StubProductFound.java

```diff
@@ -0,0 +1,11 @@
+package ca.jbrains.pos.stub;
+
+import ca.jbrains.pos.domain.Price;
+import ca.jbrains.pos.domain.ProductFound;
+
+public class StubProductFound implements ProductFound {
```

Comment:
I get nervous when I want to stub something that is just data, like a Value or an Entity. That makes me wonder whether I need to stub the Value or I would prefer to stub the _source_ of the Value. Try to write tests that give you a similar feeling of confidence without stubbing `ProductFound`/`ProductNotFound`.

Reply:
right, we should never stub the value objects, as a note, this object is being used by the PointOfSafe main function as I interpret the current requirements as a return price of 1 for any valid barcode. So I renamed the object to AnyProductWithPriceOf1

Reply:
I like that. One more step: what is special/significant about the price of 1? Consider renaming the class to describe its purpose and remove irrelevant details from the name.